### PR TITLE
Add Contributing Placeholder

### PR DESCRIPTION
**Pull request by jamesbhobbs**
Basic contributing page, will be added to as we continue to develop.

### CODE REVIEW

**Review by sareh — COMMENTED** [review]
Looks good. We should clarify our expectations for the type of contributions we would expect


Greenkeeper's Contributing.md explains process well: 
https://github.com/greenkeeperio/greenkeeper/blob/master/CONTRIBUTING.md
npm's Contributing.md explains roles within the project, e.g. community members vs employees. https://github.com/npm/npm/blob/latest/CONTRIBUTING.md

This file should link to a CODE_OF_CONDUCT.md https://www.contributor-covenant.org But we should clearly discuss how we would enforce that CoC in cases of violation before we mention a Code of Conduct in the repository.

**Comment by jamesbhobbs** [comment]
@sareh Added the code of conduct link. re adding more helpful contributions instructions, are we ready to write these? I was thinking make it plain we're open to ideas and evolve from there. Perhaps there are key points you think we should include right away?

**Comment by sareh** [comment]
Thanks for adding the CoC link.  Sure, I've added some detail.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+```
+git checkout -b <topic-branch-name>
+```
+
+When committing, please use these commit guidelines:
+
+If you have a breaking change in a commit, prefix the commit message with `BREAKING CHANGE:`
+
+### Run tests
```
Comment (jamesbhobbs) [inline CONTRIBUTING.md:57]: @sareh Should we write this and the below here and in the readme? We'll have duplication which might be missed when updating the readme, would prefer to point to a single set of instructions - probably in the readme.

**Review by sareh — DISMISSED** [review]

**Review by sareh — APPROVED** [review]

**Review by pjlee11 — APPROVED** [review]

**Pull request merged**